triframe/phases/actor.py

```diff
@@ -70,7 +74,7 @@ def prepare_history_for_actor(
                     advice = tool_use
                 message = Message(
                     content=f"<advisor>\n{advice}\n</advisor>",
-                    role="user",
+                    role="assistant",
```

Comment:
This seems like a significant behavior change. Doesn't this represent the advisor's response to the actor as if it were the actor's own output? That doesn't seem right.